npm/package.json

```diff
@@ -24,5 +24,23 @@
     "axios": "^0.18.0",
     "rimraf": "^2.6.3",
     "tar": "^4.4.8"
-  }
+  },
+  "keywords": [
+    "wrangler",
+    "cloudflare",
+    "workers",
+    "edge",
+    "compute",
+    "serverless",
+    "wasm",
+    "web",
+    "assembly",
```

Comment:
Can you put webassembly in one word and add wasm?

Reply:
👍 done :)

Reply:
I was thinking of removing web and assembly. but LGTM.